src/pages/Login/Login.jsx

```diff
@@ -1,8 +1,103 @@
 import React from 'react';
+import { useNavigate } from 'react-router-dom';
 import './Login.scss';
+import { useState } from 'react';
+import '../../components/Button/Button.scss';
+import '../../styles/variable.scss';
```

Comment:
import 순서 지켜주셔야 하고, import를 한 번에 할 수 있는 것과 import하지 않아도 되는것을 명확히 작성해주시면 좋겠습니다.
아래 예시 확인해보시죠
```suggestion
import React, { useState } from 'react';
import { useNavigate } from 'react-router-dom';
import './Login.scss';
```
1.  설명을 드리자면 `React`와 `useState`는 같은 `react` 에서 import 할 수 있습니다.
2.  Button.scss 파일을 Login 페이지에 바로 import하는 이유를 이해할 수 없습니다.
3. variable.scss은 variable.scss에 작성한 내용이 필요한 파일에서 직접 import 하셔야 합니다.

import 순서 매번 강조드리는 리뷰내용입니다. 코드작성하실 때 혹은 PR생성하셨을 때 기본적으로 체크해주세요.
간단한 내용이라도 실수가 반복되면 실수가 실력이되니까요 ^^

Reply:
반영 제대로 안되어있습니다.